### PR TITLE
Run tests and fix

### PR DESCRIPTION
**Pull request by zbynek**
On top of @halkeye 's fix

### CODE REVIEW

**Comment by timja** [comment]
why draft?

**Comment by zbynek** [comment]
@timja I needed to update one more test and wasn't sure why it started failing; turns out the dependency updates caused that, most likely https://github.com/jenkins-infra/jenkins-wiki-exporter/pull/191 . Now all is passing locally.

**Review by markyjackson-taulia — APPROVED** [review]

**Comment by zbynek** [comment]
@halkeye note that #205 still fails with
```
Scripts not permitted to use method groovy.lang.GroovyObject invokeMethod java.lang.String java.lang.Object (org.jenkinsci.plugins.docker.workflow.Docker inside java.lang.String org.jenkinsci.plugins.workflow.cps.CpsClosure2)
```
so either the Jenkinsfile needs updating or CI security settings need changing.

File: Jenkinsfile [hunk 1]
```diff
@@ -7,7 +7,11 @@ if (JENKINS_URL.contains('infra.ci.jenkins.io')) {
 if (JENKINS_URL.contains('ci.jenkins.io')) {
   node('docker&&linux') {
     checkout scm
-    sh "docker build ."
+    sh "docker build -t jenkins-wiki-exporter ."
+    docker.inside('jenkins-wiki-exporter') {
```
Comment (oleg-nenashev) [inline Jenkinsfile:11]:
```suggestion
    docker.image('jenkins-wiki-exporter').inside {
```

**Comment by halkeye** [comment]
Sweet thanks for that. I'll get this.merged and cut a release

**Pull request merged**